src/private-key.ts

```diff
@@ -42,7 +42,7 @@ export function findPrivateKey (filepath?: string): Buffer | string | null {
     throw new Error('The contents of \`PRIVATE_KEY\` could not be validated. Please check to ensure you have copied the contents of the .pem file correctly.')
   }
   if (process.env.PRIVATE_KEY_PATH) {
-    return fs.readFileSync(process.env.PRIVATE_KEY_PATH)
+    return fs.readFileSync(process.env.PRIVATE_KEY_PATH, 'utf8')
```

Comment:
Just ran into this bug. Definitely needed!

Reply:
made a separate PR: https://github.com/probot/probot/pull/817